tests/libc/tests.cmake

```diff
@@ -233,7 +233,7 @@ set(LIBC_TESTS
 )
 
 # Exclude tests that fail on Clang:
-if (NOT MY_COMPILER MATCHES "CLANG")
+if (NOT (USE_CLANGW OR MY_COMPILER MATCHES "CLANG"))
```

Comment:
Just FYI the quotes around CLANG here aren't necessary.

Reply:
There are lots of instances of "CLANG" in this file which can be fixed in the future. I added without quotes.